gradle/plugins/com.liferay.ide.gradle.core/plugin.xml

```diff
@@ -17,5 +17,12 @@
             shortName="liferay-workspace"
             projectType="workspace">
       </liferayProjectProvider>
+      <liferayProjectProvider
```

Comment:
We don't need to add a provider for this. See my changes to Andy's pull here: 
https://github.com/AndyWu2015/liferay-ide/pull/5
You can see the new pattern there.